WordPress/src/main/res/menu/posts_list_layout_change.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/post_menu_item_view_layout_type"
+        android:icon="@drawable/ic_view_post_compact_white_24dp"
+        android:title=""
```

Comment:
I think we probably should add a title to the menu item - it's useful especially for users who use accessibility tools such as TalkBack. Wdyt?